### PR TITLE
test(message): add missing tests

### PR DESCRIPTION
**Pull request by ayntee**
All tests are passing

### CODE REVIEW

File: tests/05_message.ts [hunk 1]
```diff
@@ -29,3 +39,50 @@ Deno.test({
...
+  fn() {
+    unpin(tempData.channelID, tempData.messageID);
+  },
+  ...defaultTestOptions,
+});
+
+Deno.test({
+  name: "[message] add a reaction to a message",
+  fn() {
+    addReaction(tempData.channelID, tempData.messageID, "👍");
```
Comment (Skillz4Killz) [inline tests/05_message.ts:75]: Please test the <:name:id> version also somehow

File: tests/05_message.ts [hunk 2]
```diff
@@ -29,3 +39,50 @@ Deno.test({
     assertEquals(message.embeds[0].title, "Discordeno Test");
   },
 });
+
+Deno.test({
+  name: "[message] pin a message in a channel",
+  fn() {
+    pin(tempData.channelID, tempData.messageID);
```
Comment (Skillz4Killz) [inline tests/05_message.ts:46]: We should also test this through getters imo.
`message.pin(channelID, anotherMessge.id)`

Comment (Skillz4Killz) [reply tests/05_message.ts:46]: getters should be tested for everything as well

File: tests/05_message.ts (hunk 1 above)
Comment (Skillz4Killz) [reply tests/05_message.ts:75]: We should also test addReactions

**Review by Skillz4Killz — APPROVED** [review]

File: tests/05_message.ts (hunk 2 above)
Comment (ayntee) [reply tests/05_message.ts:46]: @Skillz4Killz, the getter is basically calling `pin()` internally.

File: tests/05_message.ts (hunk 1 above)
Comment (ayntee) [reply tests/05_message.ts:75]: @Skillz4Killz, again, the `addReactions()` function `Promise.all()`s the `addReaction()` function.

**Pull request merged**